openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -473,9 +407,6 @@ def _make_jira_issue(self) -> None:
         Make our desired Jira issue.
```

Comment:
Shouldn't this whole function go away?  What's the reason for that,  I don't quite see how you've sliced the code.